UnityProject/Assets/Scripts/Systems/Research/LaserPuzzle/LaserProjection.cs

```diff
@@ -203,6 +203,13 @@ public void CleanupAndDestroy(bool Reshoot = false)
 		{
 			Destroy(Line.gameObject);
 		}
+
+		if (CustomNetworkManager.Instance._isServer)
```

Comment:
you know you can just do `CustomNetworkManager.IsServer` instead of this, right?